### DISCUSSION TITLE
"Restore Default Shortcuts" ignores Window Throw and Long Throw. Intentional?

### DISCUSSION THREAD

**Discussion by adamshackleford** [rxhanson/RectanglePro-Community#48] [Q&A]
"Settings > Restore Default Shortcuts" does not affect "Window Throw > Window Throw > Configure" or "Window Throw > Long Throw > Configure". Is this intentional?

I think it would be nice to reset them.

**Comment by rxhanson** [comment]
Good point. I will add this to my queue for the next release.

**Reply by adamshackleford** [reply]
Thank you.
